authority-portal-frontend/src/app/shared/components/common/logout-button/logout-button.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="fixed right-20 top-20">
+<div class="">
```

Comment:
class="" can be removed

Reply:
my bad, was testing and forgot to delete :)